objects/src/transaction/executed_tx.rs

```diff
@@ -124,6 +124,12 @@ impl ExecutedTransaction {
         &self.advice_witness
     }
 
+    /// Returns a reference to the transaction measurements which are the cycle counts for 
+    /// each stage
+    pub fn tx_measurements(&self) -> &TransactionMeasurements {
```

Comment:
nit: I would add a period at the end of the comment.

Also, maybe I'd name the method just `measurements()` (not sure `tx_` prefix adds much here).